frontend/src/pages/Projects/ProjectTribute.tsx

```diff
@@ -115,14 +118,24 @@ export default function ProjectTribute() {
         maxWidth: 400
       }}
     >
+      {isDisconnected && (
+        <Alert severity="error" sx={{ overflow: 'hidden', wordBreak: 'break-word' }}>
+          Your wallet has been disconnected. Click{' '}
+          <Link onClick={openConnectModal} color={'#0000EE'} sx={{ cursor: 'pointer' }}>
+            here
+          </Link>{' '}
+          to connect again.
+        </Alert>
+      )}
+
       {!isManager && (
         <Alert severity="error" sx={{ overflow: 'hidden', wordBreak: 'break-word' }}>
           You are not the manager of this project. Your wallet account "{userAddress}" does not match the manager
```

Comment:
@cptnanon do we need to add `&& !isDisconnected` to this Alert's conditional rendering logic?

Reply:
Are you suggesting to use the `userAddress` as the condition?

Reply:
Either that or test with isDisconnected. Line 131 would become:
```typescript
{!isManager && !isDisconnected (
```

That should prevent rendering empty userAddress, which makes the message confusing

> Your wallet account "" does not match the manager

See these screenshots:
https://github.com/SporosDAO/sweat-token/pull/142#issuecomment-1256254383